labgrid/resource/modbusrtu.py

```diff
@@ -1,29 +0,0 @@
-import attr
-
-from ..factory import target_factory
-from .common import Resource
-from .base import SerialPort
-
-
-@target_factory.reg_resource
-@attr.s(eq=False)
-class ModbusRTU(SerialPort, Resource):
```

Comment:
This resource should still be available, issue a deprecation warning and use the new resource automatically.

Reply:
Of course, this should be fixed. I also adjusted the commit messages to (hopefully) match the style used in labgrid